Allow advertised host override via environment variable for dev-standalone and possibly other images.

That would be very convenient for local testing and I’f bet for other scenarios too.

There's an issue with that:
```
./script_runner.sh: set: line 4: can't access tty; job control turned off
[8] 2020/08/19 06:08:49.469671 [INF] Starting nats-server version 2.1.7
[8] 2020/08/19 06:08:49.469706 [INF] Git commit [not set]
[8] 2020/08/19 06:08:49.469966 [INF] Starting http monitor on 0.0.0.0:8222
[8] 2020/08/19 06:08:49.470018 [INF] Listening for client connections on 0.0.0.0:4222
[8] 2020/08/19 06:08:49.470023 [INF] Server id is NDWTJXHEX3NCGIPUAOWBORIAZFDDM5SOSUBMACMVYWEKCPSGOLT5G3XA
[8] 2020/08/19 06:08:49.470024 [INF] Server is ready
[8] 2020/08/19 06:08:49.470242 [INF] Listening for route connections on 0.0.0.0:6222
waiting nats server
time="2020-08-19 06:08:50" level=info msg="Shutting down..."
panic: failed starting listener: listen tcp: lookup registry on 10.0.2.3:53: no such host

goroutine 1 [running]:
main.main()
	/Users/romanuseinov/projects/go/liftbridge/main.go:25 +0x1bc
./script_runner.sh: fg: line 14: job %1 not created under job control
```

In order for this to work it needs to stop looking up the advertised host, because in our case it would be an `etc/hosts` alias to docker host. I guess we could either introduce a flag for that or prevent it from looking up by default, because my guess is it should be poking at `listen` address and leave the `host` alone.

--- a/docker/dev-standalone/Dockerfile
+++ b/docker/dev-standalone/Dockerfile
@@ -20,6 +20,7 @@ EXPOSE 9292 4222 8222 6222
 VOLUME "/tmp/liftbridge/liftbridge-default"
 
 COPY docker/dev-standalone/nats-server.conf nats-server.conf
+COPY docker/dev-standalone/liftbridge.yaml liftbridge.yaml
 COPY docker/dev-standalone/script_runner.sh script_runner.sh
 
 RUN chmod +x script_runner.sh

--- a/docker/dev-standalone/README.md
+++ b/docker/dev-standalone/README.md
@@ -39,6 +39,12 @@ time="2019-08-21 11:26:14" level=info msg="Starting server on :9292..."
 time="2019-08-21 11:26:15" level=info msg="Server became metadata leader, performing leader promotion actions"
 ```
 
+If you want to advertise a docker host that is not localhost:
+```
+docker run -d --add-host registry:0.0.0.0 --name=liftbridge-main -p 4222:4222 -p 9292:9292 -p 8222:8222 -p 6222:6222 -eLIFTBRIDGE_HOST=registry liftbridge/standalone-dev
+```
+
+
 ### Volume
 
 Optionally you can specify the mount point with:

--- a/docker/dev-standalone/liftbridge.yaml
+++ b/docker/dev-standalone/liftbridge.yaml
@@ -1,0 +1,2 @@
+---
+host: localhost

--- a/docker/dev-standalone/script_runner.sh
+++ b/docker/dev-standalone/script_runner.sh
@@ -9,6 +9,6 @@ nats-server -c nats-server.conf &
 while ! nc -z localhost 8222; do sleep 1 ; echo 'waiting nats server' ; done
 
 # Start liftbridge
-liftbridge --raft-bootstrap-seed
+liftbridge --raft-bootstrap-seed -c liftbridge.yaml
 
 fg %1

